### PR TITLE
Monitoring jobs

### PR DESCRIPTION
**Pull request by sharifsalah**
Adds an introductory how-to guide for monitoring jobs using Cuetopia plugins aimed at artists / end users.

Also updates relevant navigation links.

Staged at:

https://5cd47678f6fef10008a2c339--elated-haibt-1b47ff.netlify.com/docs/user-guides/monitoring-your-jobs/

### CODE REVIEW

File: content/en/docs/User guides/monitoring-your-jobs.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+1.  Start CueGUI.
+
+    {{% alert title="Note" color="info"%}}The instructions for running
+    Cuetopia vary depending on the installation method. To learn more, see
+    [Installing CueGUI](/docs/getting-started/installing-cuegui/)
+    or contact your OpenCue admin.{{% /alert %}}
+
+    After CueGUI loads for the first time, you might see one or more blank
+    windows, as illustrated by the following screenshot:
```
Comment (bcipriano) [inline content/en/docs/User guides/monitoring-your-jobs.md]: The blank window shouldn't be a problem anymore, hopefully. There was a bug that was preventing the default window layout from being read in if an existing one wasn't found. We didn't know that wasn't the intended behavior!

Comment (bcipriano) [reply content/en/docs/User guides/monitoring-your-jobs.md]: With that in mind I suspect you might want to rework this first section a bit. Sorry about that.

Comment (sharifsalah) [reply content/en/docs/User guides/monitoring-your-jobs.md]: No worries, I should have tried to test with a fresh install but I'm finding it tricky to be certain that I'm resetting my test environment. Is there a way to reset CueGUI to default so I can see what the windows look like on first launch? (e.g. delete a config file) I'm assuming the window layout is local to the workstation and not stored server-side?

Alternatively, are you able to share a screenshot or describe what the initial layout (list of plugins) looks like?

Comment (bcipriano) [reply content/en/docs/User guides/monitoring-your-jobs.md]: If you remove `~/.config/.cuecommander/config.ini` that should do it. On next startup CueGUI will generate some log messages like:

```
WARNING   Main       Not found: /Users/cipriano/.config/.cuecommander/config.ini
Copying:   cuegui/config/cuecommander.ini
```

Which show that it is initializing the default config for you.

**Comment by sharifsalah** [comment]
Thanks, I've followed your instructions and reset CueGUI. I've updated the guide and the latest changes are staged at:

https://5cee9db081a281000aa2f1c1--elated-haibt-1b47ff.netlify.com/docs/user-guides/monitoring-your-jobs/

**Pull request merged**